packages/subgraph/manifest/data/polygon-staging.json

```diff
@@ -0,0 +1,40 @@
+{
+    "network": "matic",
+    "dataSources": {
+      "Organizations": [],
+      "OrganizationTemplates": [
+        {
+          "address": "0x64A35Cafb5FE1f70F4DF29A9bC550e93a11369F9",
+          "name": "GardensTemplate",
+          "startBlock": 21939075
+        },
+        {
+          "address": "0x9e18a165995a69e45bc4adff7795c6ca032048a7",
+          "name": "GardensTemplateWithModVotingFixedApproved",
+          "startBlock": 23936517
+        }
+      ]
+    },
+    "1HiveGarden": "0x0000000000000000000000000000000000000000",
+    "1HivePriceOracle": "0x0000000000000000000000000000000000000000",
```

Comment:
Note the strange error we were having on the polygon subgraph was related to not setting these to the zero address. A runtime error was happening here: https://github.com/1Hive/gardens/blob/master/packages/subgraph/src/constants.template.ts#L8 because the values were not defined and we end up with `Address.fromString('')` that was not recognized as an address.